dev/source/docs/gsoc-ideas-list.rst

```diff
@@ -62,21 +65,35 @@ Precision Landing accuracy improvements for Multicopters
 ArduPilot supports `Precision Landing <https://ardupilot.org/copter/docs/precision-landing-with-irlock.html>`__ using the IRLock sensor or companion computer.  This project could involve:
 
 - Add support for the `JeVois camera <http://www.jevois.org/>`__ (or similar) to recognise AprilTags and then provide the target to the autopilot via mavlink
-- Improve the reliability and accuracy of the small EKF used to estimate the landing target's relative position and velocity.  In particular adding protection against large lag in the readings leading to unreasonable results.
+- Improve the reliability and accuracy of the small EKF used to estimate the landing target's relative position and velocity. In particular adding protection against large lag in the readings leading to unreasonable results.
+- Add a landing state machine for landing retry and failure handling
 
 Lane following or visual follow-me for Copter, Rover or Boat
 ------------------------------------------------------------
 
 This project involves using machine vision and/or machine learning to add lane following or visual follow-me to ArduPilot's Copter or Rover firmware
 
-- Either a low-cost `JeVois camera <http://www.jevois.org/>`__ or a high powered `companion computer <https://ardupilot.org/dev/docs/companion-computers.html>`__ like the NVidia TX2 could be used
+- Either a low-cost camera or a high powered `companion computer <https://ardupilot.org/dev/docs/companion-computers.html>`__ like the NVidia TX2 could be used
```

Comment:
As mentioned already I'd prefer to keep the specific mention of the jevois camera